geoprocessing/src/utils/file.utils.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * File Utils
+ */
+
+import { MulterOptions } from '@nestjs/platform-express/multer/interfaces/multer-options.interface';
+import { createReadStream, readdirSync } from 'fs';
+
+/**
+ * ES5 import to avoid TS complaining
+ */
+const multer = require('multer');
+
+/**
+ * Options for Multer
+ */
+export const uploadOptions: MulterOptions = {
```

Comment:
Consider using `Object.freeze(...)` to avoid mutations (respective type of `Readonly<MulterOptions>` would be helpful as well)

Reply:
This was nice.

Fixed [here](https://github.com/Vizzuality/marxan-cloud/pull/161/commits/05d64f5ac08039cd5cef8d5b626f8418762d840d)